src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        return new Ball(new ColorSupplier().getRandomColor().toString(), new Random().nextInt(100));
```

Comment:
Don't create `Random` here, move it to the class-level declaration. Same for `CollorSupplier`.

Also, don't use `toString()` for enum, instead use `name()`.

Move magic number 100 to a constant declaration.